installers/sasview.spec

```diff
@@ -17,6 +17,7 @@ datas = [
     ('../src/sas/qtgui/Utilities/Reports/report_style.css', 'sas/qtgui/Utilities/Reports'),
     ('../src/sas/qtgui/Perspectives/Fitting/plugin_models', 'plugin_models'),
     ('../src/sas/qtgui/Utilities/WhatsNew/messages', 'sas/qtgui/Utilities/WhatsNew/messages'),
+    ('../src/sas/qtgui/Utilities/WhatsNew/css/style.css', 'sas/qtgui/Utilities/WhatsNew/messages/css'),
```

Comment:
The error seen by @smk78 is from here. The installed version is looking in `WhatsNew/messages/css` for the css file, but `WhatsNew.py` is looking in `WhatsNew/css` which doesn't exist in the installed version. Either change this line, or move the css folder into the messages path and change the the line in the python file.